material-overrides/assets/stylesheets/polkadot.css

```
@@ -889,16 +1019,30 @@ input.md-search__input::placeholder {
 
 /* Styling for nav items that are active or hovered over on smaller screens */
 @media screen and (max-width: 76.2344em) {
-  .md-nav--primary .md-nav__title[for="__drawer"] {
+  .md-nav--primary .md-nav__title .md-logo img {
+    display: none;
+  }
+
+  [data-md-color-scheme='default']
+  .md-nav--primary .md-nav__title .md-logo img[src$='#only-light'] {
+    display: inline-block;
+  }
+
+  [data-md-color-scheme='slate']
+  .md-nav--primary .md-nav__title .md-logo img[src$='#only-dark'] {
+    display: inline-block;
+  }
```

```suggestion
  [data-md-color-scheme='default']
  .md-nav--primary .md-nav__title .md-logo img[src$='#only-light'],
  [data-md-color-scheme='slate']
  .md-nav--primary .md-nav__title .md-logo img[src$='#only-dark'] {
    display: inline-block;
  }
```

can combine these into one